src/styles/blocks/main.css

```diff
@@ -0,0 +1,3 @@
+body {
+  margin: 0;
+}
```

Comment:
Consider separating styles for different BEM blocks into their own files as per the task requirements. This will help in maintaining a clean and organized codebase.